setup.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```

Comment:
On Thu, 09 Jul 2015, Lee Clemens wrote:
>  > @@ -1,4 +1,3 @@
>  > -#!/usr/bin/python

>    I'm not sure what purpose it serves (do you execute setup.py as a script -
>    not as python setup.py install)?

yes, ./setup.py  install   or ./setup.py test
it is relatively common to have it executable with a shebang

> But it does have executable bit set, and
>    actually didn't solve the problem I uncovered with the wrong python
>    executing it - so I put it back in. Ended up moving to pip install
>    instead, since that retains metadata for people to uninstall (if installed
>    from source, which we commonly recommend).

if you need to executed it with custom/other python, then you indeed
just do "custom/python setup.py", but if default is ok -- ./setup.py
should work as well... so let's just keep it as is for nowbeing.

-- 
Yaroslav O. Halchenko, Ph.D.
http://neuro.debian.net http://www.pymvpa.org http://www.fail2ban.org
Research Scientist,            Psychological and Brain Sciences Dept.
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik